src/com/xilinx/rapidwright/rwroute/PartialRouter.java

```diff
@@ -853,7 +855,8 @@ public static void main(String[] args) {
 
         // Reads in a design checkpoint and routes it
         String[] rwrouteArgs = Arrays.copyOfRange(args, 2, args.length);
-        Design routed = routeDesignWithUserDefinedArguments(Design.readCheckpoint(args[0]), rwrouteArgs);
+        Design d = Design.readCheckpoint(args[0]);
+        Design routed = routeDesignWithUserDefinedArguments(d, rwrouteArgs);
```

Comment:
```suggestion
        Design routed = routeDesignWithUserDefinedArguments(Design.readCheckpoint(args[0]), rwrouteArgs);
```
I think we can revert this, just a leftover change from debugging.

Reply:
Yeah you're right I'll revert Partial Router completely